src/randoop/operation/ConstructorCall.java

```diff
@@ -22,21 +21,21 @@
 import randoop.util.Util;
 
 /**
- * ConstructorCall is an {@link Operation} that represents a call to a 
```

Comment:
These whitespace changes are all good, but they bloat and clutter the diffs.
If your editor is configured to remove all whitespace from the ends of lines, then in order to avoid these undesirable whitespace-only diffs, could you remove all trailing whitespace from all files in Randoop and commit that to the repository once and for all?